### PR TITLE
Updated README for 2.0 pre

### PR DESCRIPTION
**Pull request by randomJoe211**
Added 2.0 installation guide and optimized the whole topic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,112 +1,108 @@
...
+   * For **Nebula Graph** 2.0 pre:
 
-The following information which indicates that you successfully connect to **Nebula Graph** is displayed:
+    ```shell
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly -u <user> -p <password> --address=graphd --port=3699
+    ```
 
-```shell
-Welcome to Nebula Graph (Version 5d10861)
+    >**NOTE**: By default, the authentication is disabled, and the `-u` and `-p` options are unnecessary. To enbale authentication, see [Authentication Configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/4.account-management-statements/authentication/#authentication).
```
Comment (oldLady344) [inline README.md:93]: keep the "Note" capitalization consistent.

File: README.md [hunk 2]
```diff
@@ -1,112 +1,108 @@
...
-```
+    ```shell
+    Welcome to Nebula Graph (Version 5d10861)
 
-**Note**: Now, you can start using **Nebula Graph** by creating spaces, tags and more. For details, refer to [get started](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/1.get-started/).
+    (user@127.0.0.1) [(none)]>
+    ```
 
-## Checking the Status and Ports of **Nebula Graph** Services
+## Check the **Nebula Graph** Service Status and Ports
```
Comment (oldLady344) [inline README.md:103]: Do we need to use the emphasis style for Nebula Graph in a heading?

File: README.md [hunk 3]
```diff
@@ -154,15 +150,15 @@ nebula-docker-compose/
         `- graph
 ```
 
-## Stopping the Services of **Nebula Graph**
+## Stop the **Nebula Graph** Services
```
Comment (oldLady344) [inline README.md:153]: Do we need to use the emphasis style for Nebula Graph in a heading?

File: README.md (hunk 2 above)
Comment (oldLady344) [inline README.md:103]: when the nebula graph services are started, users can do this check, so I think a link to this part should be added to the "deploy" part. 

this check could help new users know why the 3699 port is used for nebula console connection.

File: README.md [hunk 4]
```diff
@@ -154,15 +150,15 @@ nebula-docker-compose/
...
 
-You can stop the services of **Nebula Graph** with the following command:
+You can stop the **Nebula Graph** services with the following command:
```
Comment (oldLady344) [inline README.md:155]: If you want to stop ... , run the following command. 

the action word "run" is always recommended.

File: README.md [hunk 5]
```diff
@@ -154,15 +150,15 @@ nebula-docker-compose/
...
 
 ```shell
 $ docker-compose down -v
 ```
 
-The following information which indicates you successfully stop the services of **Nebula Graph** is displayed:
+The following information indicates you have successfully stopped the **Nebula Graph** services:
```
Comment (oldLady344) [inline README.md:161]: indicates that you have 

conjunction could make the sentence more friendly.

File: README_zh-CN.md [hunk 6]
```diff
@@ -1,136 +1,116 @@
...
+  </a>
+  <a href="http://githubbadges.com/fork.svg?user=vesoft-inc&repo=nebula&style=default">
+    <img src="http://githubbadges.com/fork.svg?user=vesoft-inc&repo=nebula&style=default" alt="nebula fork"/>
+  </a>
+  <a href="https://codecov.io/gh/vesoft-inc/nebula">
+    <img src="https://codecov.io/gh/vesoft-inc/nebula/branch/master/graph/badge.svg" alt="codecov"/>
+  </a>
+</p>
+
+## 使用 Docker Compose 部署 **Nebula Graph**
```
Comment (oldLady344) [inline README_zh-CN.md:23]: the emphasis style

File: README_zh-CN.md [hunk 7]
```diff
@@ -1,136 +1,116 @@
...
-```
+## 部署流程
 
-3. 启动所有 **Nebula Graph** 服务。
+1. 使用 Git 将 `nebula-docker-compose` 库拷贝到您的主机。
 
-```shell
-$ docker-compose up -d
-```
+   * 安装 **Nebula Graph** 1.0版本请拷贝 `master` 分支。
```
Comment (oldLady344) [inline README_zh-CN.md:41]: space between english and chinese

File: README_zh-CN.md [hunk 8]
```diff
@@ -1,136 +1,116 @@
...
+2. 进入 `nebula-docker-compose` 目录。
 
-国内从 Docker Hub 拉取镜像有时会遇到困难，此时可以配置加速镜像。例如:
+    ```shell
+    $ cd nebula-docker-compose/
+    ```
 
-* Azure 中国镜像 https://dockerhub.azk8s.cn
-* 七牛云 https://reg-mirror.qiniu.com
+    >**注意**：如果您在该主机上使用 Docker Compose 部署过其它版本的 **Nebula Graph**，为了避免兼容性问题，请先备份需要保留的[服务数据](#查看服务数据与日志)，然后删除`nebula-docker-compose/data`目录，再继续下一步。
```
Comment (oldLady344) [inline README_zh-CN.md:59]: change its position if the english version is modified.

File: README_zh-CN.md [hunk 9]
```diff
@@ -1,136 +1,116 @@
...
-**注意**：如果您没有 [Docker](https://docs.docker.com/install/) 的 `root` 权限，请参考 [如何设置 Docker 的 root 权限](https://docs.docker.com/install/linux/linux-postinstall/)。
-
-## 要部署的 **Nebula Graph** 服务
-
-在本指南中我们将部署以下 **Nebula Graph** 服务：
+* 已在主机中安装了[Docker](https://docs.docker.com/engine/install/)、[Docker Compose](https://docs.docker.com/compose/install/) 和 [Git](https://git-scm.com/download/linux)。
 
-* 3 副本 `nebula-metad` 服务
-* 3 副本 `nebula-storaged` 服务
-* 1 副本 `nebula-graphd` 服务
```
Comment (oldLady344) [inline README_zh-CN.md:25]: keep this list if the english version is modified

File: README_zh-CN.md [hunk 10]
```diff
@@ -1,136 +1,116 @@
...
 
-b. 如果您之前拉取了 `vesoft/nebula-console` 镜像，使用以下命令先删除该镜像然后再拉取：
+4. 连接到 **Nebula Graph**。
 
-   * `docker rm $(docker ps -qa -f status=exited) # cleanup exited containers`
-   * `docker rmi vesoft/nebula-console:nightly`
+    Nebula-console 是 **Nebula Graph** 的原生命令行客户端。Docker会按下方命令中指定的 Docker Hub 路径将 nebula-console 镜像文件下载到本地主机，并用其连接 **Nebula Graph** 的 graphd 服务。
 
-c. 如果您之前拉取了 **Nebula Graph** 镜像，您可以通过以下命令更新镜像：
+   * 连接到 **Nebula Graph** 1.0版本：
```
Comment (oldLady344) [inline README_zh-CN.md:83]: space between english and chinese

File: README_zh-CN.md [hunk 11]
```diff
@@ -1,136 +1,116 @@
...
 
-```shell
-$ docker run --rm -ti --network=host vesoft/nebula-console:nightly --addr=127.0.0.1 --port=3699
-```
+    ```shell
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net vesoft/nebula-console:v2-preview-nightly -u <user> -p <password> --address=graphd --port=3699
+    ```
 
-如果未开启权限管理开关，则使用默认用户和密码登录，此外无需指定；如果开启权限管理，则必须指定用户及密码。例如：
+    >**说明**：**Nebula Graph** 默认不开启身份验证功能，此时可以省略上述命令中的 `-u` 和 `-p` 选项。如需开启验证，请参见[身份验证](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/4.account-management-statements/authentication/)。
```
Comment (oldLady344) [inline README_zh-CN.md:95]: "说明“与”注意“的区别是什么？

默认情况下不开启验证功能，其实上面的命令里是不是省掉 -u -p 的描述比较好？在docker容器里设置这个配置，对不熟悉docker的人来说可能并没有那么容易。个人意见，可以到身份验证那个配置文件里做详细描述，并在那里提供完整的启动console的命令，这里只提供链接。

File: README_zh-CN.md [hunk 12]
```diff
@@ -1,136 +1,116 @@
...
-```
-
-**注意**: 现在您可以通过创建空间、tag、space 等操作来使用 **Nebula Graph**。 获取详细信息，请参考 [快速入门](https://docs.nebula-graph.com.cn/manual-CN/1.overview/2.quick-start/1.get-started/)。
+    (user@127.0.0.1) [(none)]>
+    ```
 
 ## 查看 **Nebula Graph** 服务状态和端口
 
-您可以在终端输入以下命令把所有 **Nebula Graph** 服务以列表的形式显示出来并查看其暴露的端口：
+您可以使用如下命令，以列表的形式展示 **Nebula Graph** 服务的详细信息，包括服务状态及端口号等：
```
Comment (oldLady344) [inline README_zh-CN.md:107]: 命令运行结果本来就是以它固有的形式展示的，在命令里并没有指定结果展示方式吧？

”您可以运行以下命令查看...”

File: README_zh-CN.md [hunk 13]
```diff
@@ -144,11 +124,11 @@ nebula-docker-compose_storaged1_1   ./bin/nebula-storaged --fl ...   Up (health:
...
 ```
 
-**注意**： 可以看到映射到 `nebula-docker-compose_graphd_1` 容器的 3699 的暴露端口是 3699。
+**说明**： **Nebula Graph** 默认通过 TCP3699端口向客户端提供服务，您可以在[网络配置](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/3.configurations/4.graph-config/#networking)中自定义端口号。
 
-## 查看集群数据和日志
+## 查看服务数据与日志
 
- **Nebula Graph** 的所有服务数据和日志分别存储在您本地目录 `nebula-docker-compose/data` 和 `nebula-docker-compose/logs`。
+ **Nebula Graph** 的服务数据和日志被持久化地保存在主机的 `nebula-docker-compose/data` 和 `nebula-docker-compose/logs` 目录中。
```
Comment (oldLady344) [inline README_zh-CN.md:131]: 应该是“本地目录”吧？

File: README_zh-CN.md [hunk 14]
```diff
@@ -200,4 +180,16 @@ Removing nebula-docker-compose_metad2_1    ... done
 Removing network nebula-docker-compose_nebula-net
 ```
 
-**注意**： 由于您的数据存储在您的本地电脑上，所以即使您停止 **Nebula Graph** 服务，数据也会保留在您的本地电脑。
+## 安装 **Nebula Graph** 的其它方法
```
Comment (oldLady344) [inline README_zh-CN.md:183]: the emphasis style

File: README_zh-CN.md [hunk 15]
```diff
@@ -200,4 +180,16 @@ Removing nebula-docker-compose_metad2_1    ... done
...
+* [使用 .rpm 或 .deb 文件](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/)
+
+## 常见问题
+
+**问**：如何更新 nebula-console 客户端？
+
+**答**：在主机的`nebula-docker-compose`目录中使用`docker-compose pull`命令即可。
+
+# 下一步
+您可以进行创建图空间和插入数据等操作，详情请参见[ **Nebula Graph** 快速入门](https://docs.nebula-graph.com.cn/manual-CN/1.overview/2.quick-start/1.get-started/)。
```
Comment (oldLady344) [inline README_zh-CN.md:195]: 您可以开始创建图数据库，详细信息，参考 []()

File: README_zh-CN.md (hunk 11 above)
Comment (randomJoe211) [reply README_zh-CN.md:95]: 说明一般分四个级别
NOTE/说明：一般性的补充说明
NOTICE/注意：可能影响操作的事项，用户需要注意，但不一定会造成严重影响
WARNING/警告：如果不正确操作会带来严重后果的注意事项，比注意的语气要强，需要用户务必小心
DANGER/禁止：提醒用户务必不要做的事

File: README.md [hunk 16]
```diff
@@ -182,9 +178,18 @@ Removing nebula-docker-compose_metad2_1    ... done
...
+
+* [Using Source Code](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code/)
+* [Using Docker](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker/)
+* [Using .rpm or .deb Files](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/)
+
+## FAQ
+
+**Q**: How to update the nebula-console client?
+
+**A**: Use the `docker-compose pull` command in the `nebula-docker-compose` directory on your host.
```
Comment (yixinglu) [inline README.md:191]: This command is to update nebula service images, not console image. The right command is like:

```
docker pull vesoft/nebula-console:nightly   # v1.0
docker pull vesoft/nebula-console:v2-preview-nightly  # v2-preview
```

File: README_zh-CN.md [hunk 17]
```diff
@@ -1,136 +1,116 @@
...
-  <br>
-</div>
-
-## 概述
-
-在本文档中，我们将指导您使用 [Docker](https://docs.docker.com/install/) 和 [Docker Compose](https://docs.docker.com/compose/install/) 来部署 **Nebula Graph** 集群。我们将向您展示如何[查看 **Nebula Graph** 的服务状态和端口](#查看-Nebula-Graph-服务状态和端口)，如何[查看集群数据和日志](#查看集群数据和日志), 以及如何[停止 **Nebula Graph** 服务](#停止-Nebula-Graph-服务)。
+<p align="center">
+  <img src="https://github.com/vesoft-inc/nebula/raw/master/docs/logo.png"/>
+  <br>中文 | <a href="README.md">English</a>
+  <br>世界上唯一能够容纳千亿个顶点和万亿条边，并提供毫秒级查询延时的图数据库解决方案<br>
```
Comment (yixinglu) [inline README_zh-CN.md:4]: Emmm.  I think it's better to delete the description of '世界上唯一'.

File: README_zh-CN.md [hunk 18]
```diff
@@ -200,4 +180,16 @@ Removing nebula-docker-compose_metad2_1    ... done
...
+## 安装 **Nebula Graph** 的其它方法
+* [使用源代码](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code/)
+* [使用 Docker](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/1.build/2.build-by-docker/)
+* [使用 .rpm 或 .deb 文件](https://docs.nebula-graph.com.cn/manual-CN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/)
+
+## 常见问题
+
+**问**：如何更新 nebula-console 客户端？
+
+**答**：在主机的`nebula-docker-compose`目录中使用`docker-compose pull`命令即可。
```
Comment (yixinglu) [inline README_zh-CN.md:192]: The same issue with my first comment.

File: README.md [hunk 19]
```diff
@@ -1,112 +1,108 @@
...
+Using Docker Compose is a convenient way to deploy **Nebula Graph** services. This topic guides you through the deployment and shows you how to [check the **Nebula Graph** service status](#check-the-nebula-graph-service-status-and-ports), [data and logs](#check-the-service-data-and-logs), and how to [stop the **Nebula Graph** services](#stop-the-nebula-graph-services).
 
 ## Prerequisites
 
-Before you start deploying the **Nebula Graph** cluster, ensure that you have installed the latest version of [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/).
+* You have installed [Docker](https://docs.docker.com/engine/install/), [Docker Compose](https://docs.docker.com/compose/install/) and [Git](https://git-scm.com/download/linux) on your host.
```
[the remark this answers is not in the document]
Comment (randomJoe211) [reply README.md:25]: Thank you and done~

File: README.md (hunk 16 above)
Comment (randomJoe211) [reply README.md:191]: Thank you for the heads-up!

File: README_zh-CN.md (hunk 17 above)
Comment (randomJoe211) [reply README_zh-CN.md:4]: Deleted~

File: README_zh-CN.md (hunk 18 above)
Comment (randomJoe211) [reply README_zh-CN.md:192]: Thank you and fixed.

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**